source/NVDAObjects/UIA/__init__.py

```diff
@@ -767,19 +767,19 @@ def _get_description(self):
 			return ""
 
 	def _get_keyboardShortcut(self):
-		ret = ""
 		try:
-			ret += self.UIAElement.currentAccessKey
+			accessKey = self.UIAElement.currentAccessKey

```

Comment:
It probably shouldn't happen, but it's possible that this could itself return None (rather than throwing an exception). Since we're being safe, we should probably handle that.